Ehr_OOP.py

```diff
@@ -0,0 +1,97 @@
+"""Bio stats assignment #OOP Mohammad Anas."""
+
+
+from datetime import datetime
+
+
+class Patient:
+    """Store a patient data."""
+
+    def __init__(self, PatientID):
+        """Instantiate Patient Class."""
+        self.patientID = PatientID
+        self.gender = None
+        self.DOB = None
+        self.race = None
+
+    def __repr__(self):
+        """Patient Class Representation."""
+        return self.patientID + " Class"
+
+    def store_info(self, patientinfo):
+        """Extract info of a patient ID from text file."""
+        self.gender = patientinfo["PatientGender"]
+        self.DOB = datetime.strptime(
+            patientinfo["PatientDateOfBirth"], "%Y-%m-%d %H:%M:%S.%f"
+        )
+        self.race = patientinfo["PatientRace"]
+
+    @property
+    def age(self):
+        """Calculate patient age."""
+        today = datetime.today()
+        return int((self.DOB - today).days / 365.25)
+
+
+class Lab:
+    """Store Lab data."""
+
+    def __init__(self, Labname):
+        """Instantiate Lab Class."""
+        self.labname = Labname
+        self.units = []
+        self.value = []
+        self.patientID = []
+
+    def __repr__(self):
+        """Lab Class Represenatation."""
+        return self.labname + " Class"
+
+    def store_info(self, lab_info):
+        """Extract Lab info."""
+        self.value.append(lab_info["LabValue"])
+        self.units.append(lab_info["LabUnits"])
+        self.patientID.append(lab_info["PatientID"])
+
+
+# redefine the function parse data that returns a class
+
+
+def parse_data(filename, pt_lb):
```

Comment:
I'd probably just make separate `parse_patient_data` and `parse_lab_data` functions.